packages/engine/src/worker/runner/python/batch.py

```diff
@@ -116,6 +118,8 @@ def __init__(self, batch_id):
         self.static_meta = None
 
     def sync(self, latest_batch, schema=None):
+        print("loaded", self.mem_version, self.batch_version, "latest",
+              latest_batch.mem_version, latest_batch.batch_version)
```

Comment:
Can we avoid print statements in Python code? This will spam the terminal.

Reply:
This was for debugging; I'll remove it entirely

Reply:
Addressed in https://github.com/hashintel/hash/pull/405